### PR TITLE
excess comma removed

### PR DESCRIPTION
**Pull request by jankoboz**
This is how i remove commas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ saveCityZipCode(address.match(cityZipCodeRegex)[1], address.match(cityZipCodeReg
 ```javascript
 const address = 'One Infinite Loop, Cupertino 95014';
 const cityZipCodeRegex = /^[^,\\]+[,\\\s]+(.+?)\s*(\d{5})?$/;
-const [, city, zipCode] = address.match(cityZipCodeRegex) || [];
+const [city, zipCode] = address.match(cityZipCodeRegex) || [];
```
Comment (adamkecskes) [inline README.md]: Have you tried this?
The first element will be the _entire matched string_.
The comma is required to be there.

Comment (jankoboz) [reply README.md]: Not I have not tried. My mistake. i was just thinking it was a typo. Sorry about that

Comment (rodgoeliz) [reply README.md]: @jankoboz The entire string is matched because of the capturing groups, "( )" in the regex  :)

**Comment by jankoboz** [comment]
Thanks for the explanation, really appreciated. I would like to translate
this project to Turkish. Do you think it is a good idea? Should i start it ?

**Comment by ryanmcdermott** [comment]
Please feel free to translate into any language you are comfortable with! Much appreciated :)

**Pull request closed without merging**